codecs/jxl/enc/jxl_enc.cpp

```diff
@@ -26,7 +26,7 @@ val encode(std::string image, int width, int height, JXLOptions options) {
   jxl::CodecInOut io;
   jxl::PaddedBytes bytes;
   jxl::ImageBundle* main = &io.Main();
-  jxl::ThreadPoolInternal *pool_ptr = nullptr;
+  jxl::ThreadPoolInternal* pool_ptr = nullptr;
```

Comment:
Huh, surprised my commit handler didn't catch this when I pushed stuff. Maybe I didn't do `npm install` lately though.